From 1edf378ba6f3603412f4e6686e651f16b3607f7a Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Fri, 19 Apr 2013 17:30:39 +0800
Subject: [PATCH 191/602] ENGR00223120 ARD:delete location test in CTS

For no GPS hardware on ARD board. so comment it here to not test it.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 sabreauto_6q/required_hardware.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sabreauto_6q/required_hardware.xml b/sabreauto_6q/required_hardware.xml
index a779336..ef9a66b 100644
--- a/sabreauto_6q/required_hardware.xml
+++ b/sabreauto_6q/required_hardware.xml
@@ -21,8 +21,8 @@
     <feature name="android.hardware.camera" />
     <feature name="android.hardware.camera.front" />
     <feature name="android.hardware.camera.flash" />
-    <feature name="android.hardware.location" />
-    <feature name="android.hardware.location.network" />
+    <!-- <feature name="android.hardware.location" /> -->
+    <!-- <feature name="android.hardware.location.network" /> -->
     <feature name="android.hardware.sensor.accelerometer" />
     <feature name="android.hardware.sensor.compass" />
     <!-- <feature name="android.hardware.bluetooth" /> -->
-- 
1.8.0

